spec/mailers/user_welcome_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe UserWelcomeMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
En PR del primer sprint recomendaron:
- No incluir estos archivos
- Incluirlos sin  `pending "add some examples to (or delete) #{__FILE__}"` para evitar conflictos en la mezcla.

Reply:
Genial! Me había olvidado.